lib/ansible/config/base.yml

```diff
@@ -1474,16 +1473,16 @@ INTERPRETER_PYTHON_DISTRO_MAP:
 INTERPRETER_PYTHON_FALLBACK:
   name: Ordered list of Python interpreters to check for in discovery
   default:
-  - /usr/bin/python
   - python3.9
   - python3.8
   - python3.7
   - python3.6
   - python3.5
+  - /usr/bin/python3
   - python2.7
   - python2.6
+  - /usr/bin/python
   - /usr/libexec/platform-python
-  - /usr/bin/python3
```

Comment:
I think the order of this list reflects the strategy:
- prefer a specific Python 3 version
- then a generic Python 3 version
- then a specific Python 2 version
- then a generic Python version (potentially either 2 or 3)
- lastly, use `platform-python`